docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -126,7 +126,7 @@ with a suite of modules that can be configured, composed, and extended to develo
 
 ## Rollups
 
-Rollups, a layer 2 scaling solution, have been talked about for some time and it's potential to scaling Ethereum. A layer 2 network of nodes would be responsible for "rolling up" transaction together by batching them before publishing them to the main Ethereum chain. A rolled up transaction could inclue 10,000 transactions. Rollups increase TPS and can reduce transaction fees significantly. When it comes to implementing it, there are a few different ways to approach the problem.
+Rollups, a layer 2 scaling solution, have been talked about for some time, and their potential to scale Ethereum. A layer 2 network of nodes would be responsible for "rolling up" transaction together by batching them before publishing them to the main Ethereum chain. A rolled-up transaction could include 10,000 transactions. Rollups increase TPS and can reduce transaction fees significantly. When it comes to implementing it, there are a few different ways to approach the problem.
```

Comment:
10,000 seems to be an arbitrary choice.

style: layer-2.

typo: "rolling up" transaction**s**

Either the transactions themselves are published _or_ a commitment to them. It's more typical for a commitment to be published to the chain and the transactions themselves to be published to a DA layer. In the case of ZK rollups, the transactions may not need to be published - only the changes to the rollup chain-state.

I would replace the last sentence with "Rollups are a broad category of solution which encompass many different possible architectures and implementations."